docs/source/blogs/tech_blog/blog12_Inference_Time_Compute_Implementation_in_TensorRT-LLM.md

```diff
@@ -0,0 +1,425 @@
+# Inference Time Compute Implementation in TensorRT LLM
+
+By NVIDIA TensorRT LLM Team
+
+## Table of Contents
+- [Inference-Time Compute Implementation in TensorRT LLM (Part 1: Design and Implementation](#inference-time-compute-implementation-in-tensorrt-llm)
+  - [Table of Content](#table-of-content)
```

Comment:
Delete this; it is redundant. The table of contents does not need to link to itself.